src/webgpu/api/validation/encoding/cmds/copyTextureToTexture.spec.ts

```diff
@@ -349,6 +349,9 @@ g.test('depth_stencil_copy_restrictions')
     } = t.params;
 
     const kMipLevelCount = 3;
+
+    await t.selectDeviceOrSkipTestCase(kAllTextureFormatInfo[format].extension);
```

Comment:
same

Reply:
Done. Sorry, I didn't look into other tests when you pointed out this problem.